pkgs/applications/window-managers/i3/wmfocus.nix

```diff
@@ -12,7 +12,7 @@ rustPlatform.buildRustPackage rec {
     sha256 = "0jx0h2zyghs3bp4sg8f3vk5rkyprz2dqfqs0v72vmkp3cvgzxbvs";
   };
 
-  cargoSha256 = "1xmc28ns59jcmnv17102s2084baxqdvi0ibbyqwb108385qnixzf";
+  cargoSha256 = "000000004cb26jc9rynhva0640xgb92wvs7ybqq4bsbb0l1xfxz6";
```

Comment:
This one looks weird. Is it correct?

Reply:
Good spot! That’s a placeholder I’d put in to test the checksums that never got replaced. I shall drop that commit since it doesn’t build anyway.